src/core/reference/include/ngraph/runtime/reference/multiclass_nms.hpp

```diff
@@ -15,18 +15,218 @@
 #include <vector>
 
 #include "ngraph/node.hpp"
+#include "ngraph/op/util/multiclass_nms_base.hpp"
 #include "ngraph/op/util/nms_base.hpp"
 #include "ngraph/op/util/op_types.hpp"
 #include "ngraph/shape_util.hpp"
 
 namespace ngraph {
 namespace runtime {
 namespace reference {
+namespace multiclass_nms_impl {
+namespace {
+std::vector<float> get_floats(const std::shared_ptr<HostTensor>& input, const Shape& shape) {
```

Comment:
If no performance consideration or something else, it's recommended to move to .cpp to hide the implementation details.

Reply:
great idea. Code refactor in need here.